Model/CategoryManagerInterface.php

```diff
@@ -0,0 +1,63 @@
+<?php
+
+/*
+ * This file is part of the Sonata Project package.
+ *
+ * (c) Thomas Rabaix <thomas.rabaix@sonata-project.org>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sonata\MediaBundle\Model;
+
+use Sonata\ClassificationBundle\Model\CategoryInterface;
+
+/**
+ * @author Joao Albuquerque <albuquerque.joao.filipe@gmail.com>
+ * @author Christian Gripp <mail@core23.de>
+ */
+interface CategoryManagerInterface
+{
+    /**
+     * @param string $context
+     *
+     * @return CategoryInterface
+     */
+    public function getRootCategory($context);
+
+    /**
+     * @param bool|true $loadChildren
```

Comment:
I'd argue that [`null` is a type](http://php.net/manual/en/language.types.null.php). `true` is not. The link you provided says it can be used to indicate when a method always returns `true`, but that's all…